src/serlo-editor/plugins/fill-in-the-blanks-exercise/blank-renderer.tsx

```diff
@@ -33,7 +33,7 @@ export function BlankRenderer(props: {
       {mode === 'fill-in-the-blanks' ? (
         <input
           className={clsx(
-            'h-full resize-none rounded-full border border-brand bg-brand-50 px-2',
+            'my-0.5 h-full resize-none rounded-full border border-brand bg-brand-50 pl-2',
```

Comment:
More space is nice but now the line spacing differs between lines with & without blanks